config/500-controller.yaml

```diff
@@ -53,9 +55,13 @@ spec:
         ports:
           - containerPort: 9090
             name: metrics
-        resources:
+        resources:  # An initial guess, but consistent with serving
+          requests:
+            cpu: 100m
+            memory: 100Mi
           limits:
-            memory: 1000Mi  # An initial guess, but consistent with serving
+            cpu: 1000m
+            memory: 1000Mi
         volumeMounts:
```

Comment:
Talked @bbrowning about this, and not exactly sure if we really need to add these...

Mantra: There is no 1 limit value that works for everyone.

Reply:
Yes, this ties in to other conversations held in Slack with @mattmoor, @raushan2016 about whether we should be setting limits for anything by default. And, if we do, what's the process we use to verify that the limits we set are appropriate? And at what scale?

If we arbitrarily choose limits without basing their values based on actual observed values tested at scale then we risk choosing limits that are too low and causing headaches as people use Knative in production.

Reply:
Totally agree with @matzew and @bbrowning. The intention was to be consistent with serving and add new requests as in [Kyma](https://github.com/kyma-project/kyma/blob/master/resources/knative-eventing/charts/knative-eventing/templates/eventing.yaml#L723). It was found the controller pod exerts 1000Mi of memory request resulting in reserving quite a lot more than it actually needs at that moment.

Reply:
@matzew @bbrowning I removed the resource related changes for now as it needs a profound investigation. Can you please check if I need to make any changes?